Caboodle/Email/Email.android.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Android;
+using Android.Content;
+using Android.OS;
+using Android.Text;
+using Android.Net;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class Email
+    {
+        /// Indicating whether Compose Dialog is available
+        public static bool IsComposeSupported
+            => Platform.IsIntentSupported(CreateIntent(
+                new EmailMessage()
+                {
+                    RecipientsTo = new string[] {"mailto:caboodle@mailinator.com" },
+                }));
+
+        /// Indicating whether Sending in background is available
+        public static bool IsSendSupported
+        {
+            get;
+        }
```

Comment:
This technically has no explicit value (default to false). Also, this must be internal until we decide to expose it.